bundles/com.amazonaws.eclipse.core/src/com/amazonaws/eclipse/core/mobileanalytics/AwsToolkitMetricType.java

```diff
@@ -29,68 +29,66 @@
-    EXPLORER_S3_DELETE_BUCKET("Explorer-S3DeleteBucket"),
-    EXPLORER_S3_DELETE_OBJECT("Explorer-S3DeleteObject"),
-    EXPLORER_S3_GENERATE_PRESIGNED_URL("Explorer-S3GeneratePresignedUrl"),
-    EXPLORER_S3_EDIT_OBJECT_TAGS("Explorer-S3EditObjectTags"),
-    EXPLORER_S3_EDIT_OBJECT_PERMISSIONS("Explorer-S3EditObjectPermissions"),
+    EXPLORER_S3_CREATE_BUCKET("s3_createBucket"),
+    EXPLORER_S3_OPEN_BUCKET_EDITOR("s3_openEditor"),
+    EXPLORER_S3_REFRESH_BUCKET_EDITOR("s3_refreshEditor"),
+    EXPLORER_S3_DELETE_BUCKET("s3_deleteBucket"),
+    EXPLORER_S3_DELETE_OBJECTS("s3_deleteObjects"),
+    EXPLORER_S3_GENERATE_PRESIGNED_URL("s3_genereatePresignedUrl"),
+    EXPLORER_S3_EDIT_OBJECT_TAGS("s3_editObjectTags"),
+    EXPLORER_S3_EDIT_OBJECT_PERMISSIONS("s3_editObjectPermissions"),
     /* Explorer EC2 Actions */
-    EXPLORER_EC2_OPEN_VIEW("Explorer-Ec2OpenView"),
-    EXPLORER_EC2_OPEN_AMIS_VIEW("Explorer-Ec2OpenAmisView"),
-    EXPLORER_EC2_OPEN_INSTANCES_VIEW("Explorer-Ec2OpenInstancesView"),
-    EXPLORER_EC2_OPEN_EBS_VIEW("Explorer-Ec2OpenEbsView"),
-    EXPLORER_EC2_OPEN_SECURITY_GROUPS_VIEW("Explorer-Ec2OpenSecurityGroupsView"),
-    EXPLORER_EC2_SELECT_SECURITY_GROUP("Explorer-Ec2SelectSecurityGroup"),
-    EXPLORER_EC2_NEW_SECURITY_GROUP("Explorer-Ec2NewSecurityGroup"),
-    EXPLORER_EC2_DELETE_SECURITY_GROUP("Explorer-Ec2DeleteSecurityGroup"),
-    EXPLORER_EC2_ADD_PERMISSIONS_TO_SECURITY_GROUP("Explorer-Ec2AddPermissionsToSecurityGroup"),
-    EXPLORER_EC2_REMOVE_PERMISSIONS_FROM_SECURITY_GROUP("Explorer-Ec2RemovePermissionsFromSecurityGroup"),
-    EXPLORER_EC2_REFRESH_SECURITY_GROUP("Explorer-Ec2RefreshSecurityGroup"),
-    EXPLORER_EC2_OPEN_SHELL_ACTION("Explorer-Ec2OpenShellAction"),
-    EXPLORER_EC2_OPEN_SHELL_DIALOG_ACTION("Explorer-Ec2OpenShellDialogAction"),
-    EXPLORER_EC2_REBOOT_ACTION("Explorer-Ec2RebootAction"),
-    EXPLORER_EC2_TERMINATE_ACTION("Explorer-Ec2TerminateAction"),
-    EXPLORER_EC2_CREATE_AMI_ACTION("Explorer-Ec2CreateAmiAction"),
-    EXPLORER_EC2_COPY_PUBLIC_DNS_NAME_ACTION("Explorer-Ec2CopyPublicDnsNameAction"),
-    EXPLORER_EC2_ATTACH_NEW_VOLUME_ACTION("Explorer-Ec2AttachNewVolumeAction"),
-    EXPLORER_EC2_START_INSTANCES_ACTION("Explorer-Ec2StartInstancesAction"),
-    EXPLORER_EC2_STOP_INSTANCES_ACTION("Explorer-Ec2StopInstancesAction"),
+    EXPLORER_EC2_OPEN_VIEW("ec2_openEditor"),
+    EXPLORER_EC2_OPEN_AMIS_VIEW("ec2_openAmis"),
+    EXPLORER_EC2_OPEN_INSTANCES_VIEW("ec2_openInstances"),
+    EXPLORER_EC2_OPEN_EBS_VIEW("ec2_openEbs"),
+    EXPLORER_EC2_OPEN_SECURITY_GROUPS_VIEW("ec2_openSecurityGroups"),
+    EXPLORER_EC2_NEW_SECURITY_GROUP("ec2_createSecurityGroup"),
+    EXPLORER_EC2_DELETE_SECURITY_GROUP("ec2_deleteSecurityGroup"),
+    EXPLORER_EC2_ADD_PERMISSIONS_TO_SECURITY_GROUP("ec2_addPermissionToSecurityGroup"),
+    EXPLORER_EC2_REMOVE_PERMISSIONS_FROM_SECURITY_GROUP("ec2_removePermissionToSecurityGroup"),
```

Comment:
Consider To -> From

Reply:
makes more sense, was just translating literally. Fixed